lms/static/sass/course/_auto-cert.scss

```diff
@@ -0,0 +1,42 @@
+.progress-certificates {
+    margin-bottom: 0 !important;
+    padding-bottom: 0 !important;
+    border-bottom: 0 !important;
+
+    .title-no-float {
```

Comment:
No style based classes please. You can also edit the h1 style in the file above since it's scoped just to this instance.